services/wallet/transfer/commands_sequential.go

```diff
@@ -144,11 +170,20 @@ func (c *findNewBlocksCommand) detectNonceChange(parent context.Context, to *big
 
 		newNonce, err := c.balanceCacher.NonceAt(parent, c.chainClient, account, to)
 		if err != nil {
-			log.Error("findNewBlocksCommand can't get nonce", "error", err, "account", account, "chain", c.chainClient.NetworkID())
+			logutils.ZapLogger().Error("findNewBlocksCommand can't get nonce",
+				zap.Stringer("account", account),
+				zap.Uint64("chain", c.chainClient.NetworkID()),
+				zap.Error(err),
+			)
 			return nil, err
 		}
 
-		log.Debug("Comparing nonces", "oldNonce", *oldNonce, "newNonce", *newNonce, "to", to, "acc", account)
+		logutils.ZapLogger().Debug("Comparing nonces",
+			zap.Any("oldNonce", oldNonce),
+			zap.Any("newNonce", newNonce),
```

Comment:
FYI, theres a poitner version for each type in zap:
```suggestion
			zap.Int64p("oldNonce", oldNonce),
			zap.Int64p("newNonce", newNonce),
```
This applies to a few more places, at least for nonces in this file.